net/tcp.lisp

```diff
@@ -645,7 +645,7 @@ Set to a value near 2^32 to test SND sequence number wrapping.")
                                       :port (tcp-connection-remote-port connection)))
                 (detach-tcp-connection connection)
                 (tcp4-send-packet connection
-                                  (tcp-connection-snd.next connection)
+                                  (tcp-connection-snd.nxt connection)
```

Comment:
This one I can approve right away because I have the fix on a local branch; I noticed the runtime issue the typo causes while I was testing the SLIP driver.